cattle/plugins/docker/compute.py

```diff
@@ -716,3 +716,24 @@ def _do_instance_remove(self, instance, host, progress):
             return
 
         client.remove_container(container, force=True, v=True)
+
+    def _do_instance_inspect(self, instanceInspectRequest):
+        client = docker_client()
+        container = None
+        try:
+            container_id = instanceInspectRequest.id
+            container = self.get_container_by(client, lambda x:
+                                              self._id_filter(container_id, x))
+        except (KeyError, AttributeError):
+            pass
+
+        try:
+            name = '/{0}'.format(instanceInspectRequest.name)
+            container = self.get_container_by(client, lambda x:
```

Comment:
If the container was already found by the id, do we want to intentionally override it?